web/src/app/components/job-list-item/job-list-item.component.spec.ts

```diff
@@ -238,4 +251,32 @@ describe('JobListItemComponent', () => {
     // Expect 6 nodes since one is for the loi and 5 for submissions
     expect((await job.getNodes()).length).toBe(6);
   });
+
+  it('should highlight LOI when it is selected', () => {
+    const lois = createLois(1);
+    lois$.next(lois);
+
+    locationOfInterestId$.next(lois.first()!.id);
+    fixture.detectChanges();
+
+    const loiNodeElement = fixture.debugElement.query(By.css('mat-tree-node'));
```

Comment:
It's best to use the tree test harness here instead. The following should work instead:
```
const loiNodeElement = await loader.getHarness(MatTreeNodeHarness);
expect (await (await loiNodeElement.host()).hasClass('tree-node-selected');
```

Reply:
This is exactly what I needed, and I don't need that `fixture.detectChanges();` no more. Great tip! Thanks! Just had to add that `toBeTrue()` at the end, otherwise jasmine gives me a warning.